Gemfile

```diff
@@ -73,7 +73,8 @@ gem 'acts_as_list', '~> 0.9.17'
 gem 'braintree', '~> 2.93'
 gem 'bugsnag', '~> 6.11'
 gem 'cancancan', '~> 2.3.0'
-gem 'formtastic', '~> 2.3.1'
+gem 'formtastic', '~> 4.0'
+gem 'gruff', '~>0.3', require: false
```

Comment:
Can you please rebase on master or merge master and make sure `gruff` is not included?

It was removed here https://github.com/3scale/porta/pull/3165/files